src/consent-manager/container.js

```diff
@@ -167,6 +167,10 @@ export default class Container extends PureComponent {
     saveConsent(undefined, false, false)
   }
 
+  noImpliedConsentAnchorTapped = e => {
+    return e.target.tagName === 'A' && e.target.dataset.implyConsent === 'false'
```

Comment:
You would have to use `closest` like this, otherwise the `target` may not be an `a` tag

https://github.com/healthbridgeltd/consent-manager/pull/10/files#diff-50987a3a3d7742f99710e9fbd9693835R196

Reply:
This shouldn't be an issue as the target is always the anchor tag that bubbles up to the body which captures the event. However, if in the future we want to add the `data-imply-consent="false"` to other elements then I would rather that we used:

`e.target.closest([data-imply-consent="false"])`

Reply:
Could we closest polyfill, otherwise it would fail in IE11? We have polyfill for closest in zavamed, but we don't have it assessment, possibly in other apps.

Reply:
Sure - we already use `closest()` elsewhere, but we can add a transform during linting to handle it for us.

Reply:
Yeah, nevermind, we have closest polyfill everywhere and we were already using `closest` in consent-manager